src/Pages/form/Components/GoToTopButton.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { createStyles, makeStyles } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+
+const useStyle = makeStyles((theme) =>
+  createStyles({
+    scrollToTop: {
+      display: "block",
+      marginLeft: "auto",
+      fontSize: "25px",
+      border: "none",
+      outline: "none",
+      backgroundColor: "#EF5DA8",
+      color: "white",
+      cursor: "pointer",
+      padding: "15px",
+      borderRadius: "50%",
+      height: "60px",
+      width: "50px",
+      justifyContent: "center",
```

Comment:
JustifyContenr and alignItems are not working because they should be used in display flex.